Sprint-2/errors/2.js

```diff
@@ -7,4 +7,12 @@ function square(3) {
     return num * num;
 }
 
+The reason we are getting an error is because our function is very confusing,  the square function parenthesis  has a value of 3 which was not declared.
+ Also we cannot have a value in the parenthesis because we want a function that will square any number that we declare, here is the fixed function below 
```

Comment:
We cannot have this value and we do want a function that squares any number, but are these two statements related? _Why_ can we not have a number as a parameter in JavaScript, precisely?

Reply:
I have refined my answer and I think one of the reasons we cannot have a number as a parameter is because a number is not flexible but if we use placeholders then our function becomes flexible and we can use it in different cases.